### ISSUE TITLE
index bullets are inconsistent on the docs site home page

### ISSUE THREAD

**Issue by SanyaKochhar** [openservicemesh/osm-docs#272]
tl;dr: on docs site home page, sections and subsections are not distinctive and all section titles after guides have bullets they are not supposed to.

In general, when you navigate to a page other than home page, the docs site index is split distinctly into headings and subheadings. Subheads are bulleted like so:
![image](https://user-images.githubusercontent.com/42152676/140181994-8a780df1-c10e-433d-a3f9-e7aa915c25da.png)

But when you go to the docs site home page, the bullets become inconsistent and the headers are also bulleted. It is hard to tell which are section titles and which are subsections:
![image](https://user-images.githubusercontent.com/42152676/140182122-b2522796-a421-4f34-b9e9-3f320fb9a9e1.png)

**Comment by allenlsy** [comment]
This is resolved with new theme.
